_layouts/styleguide.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="en">
+<!--[if lt IE 9]><html class="lt-ie9"><![endif]-->
```

Comment:
@msecret As of right now, yes, it is just for the styleguide. But when we get into bug testing, we may need this again as well.

Why would we need to tell users they need it to get the library working? I don't think they'd need this information. None of the components require it.